config/initializers/simple_form.rb

```diff
@@ -2,6 +2,10 @@
 SimpleForm.setup do |config|
   require Rails.root.join('lib', 'extensions', 'simple_form', 'error_notification')
   require Rails.root.join('lib', 'extensions', 'simple_form', 'components', 'submit_component')
+  require Rails.root.join(
+    'lib', 'extensions', 'simple_form', 'components',
+    'html5_no_aria_required'
```

Comment:
I might suggest the following style toward keeping either one entry per line or all entries on a single line. I find it hard to scan when there are mixed numbers of entries across lines.

```suggestion
    'lib', 'extensions', 'simple_form', 'components', 'html5_no_aria_required'
```

Reply:
Fixed that so it's all on one but passes lint.